certified-connectors/Powell Teams/README.md

```diff
@@ -0,0 +1,24 @@
+# Powell Teams
+
+Powell Teams API endpoints documentation.

```

Comment:
@PowellSoftware Please provide a valid description of your connector which everyone can understand

Reply:
Ok, we just updated the description in the readme file.
=>
Powell Teams API endpoints allows you to manage the governance of your teams (creation / update of your teams).

Reply:
We put the initial code which had been validated previously. Can you confirm that this is right for you?
=> apiDefinition.swagger.json

`"swagger": "2.0",
  "info": {
    "title": "Powell.Teams.Web.PublicAPI",
    "description": "Powell Teams API endpoints documentation.",
    "version": "V1.1",
    "contact": {
      "name": "Powell Teams support",
      "url": "https://support.powell-software.com/hc/en-us/requests/new",
      "email": "support-teams@powellsoftware.zendesk.com"
    }`